macpython-download-cache-and-build-module-wheels.sh

```diff
@@ -1,12 +0,0 @@
-#!/bin/bash
```

Comment:
No it shouldn't be here, it the script downloaded and used by travisCI to generate the python wheels for mac. It accidentally had been added in a previous commit.